package.json

```diff
@@ -32,7 +32,7 @@
     "@types/styled-components": "^5.1.10",
     "@types/styled-system": "^5.1.11",
     "@types/styled-system__css": "^5.0.15",
-    "@urbit/roller-api": "^0.1.8",
+    "@urbit/roller-api": "^0.1.9",
```

Comment:
Since this was addressed in #641, does this branch need to be rebased on `roller`?

Reply:
I'll look locally if there's any conflicts, although the merge message here says that all is fine